package com.example.veterinaria.Repository;

import com.example.veterinaria.Entity.HistoriaClinica;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface HistoriaClinicaRepository extends JpaRepository <HistoriaClinica, Long> {
}
